queries/usd/locals.scm

```diff
@@ -0,0 +1,4 @@
+[
+  (block)
+  (metadata)
```

Comment:
Just a question, but should multiline tuples, arguments, ... be folded as well? And if so, should it be folded with the block, or not